components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/utils/AttributeMapper.java

```diff
@@ -245,16 +247,9 @@ private static void setClaimsForComplexAttribute(Attribute entry, Map<String, St
 
     }
 
-    /**
-     * Construct the SCIM Object given the attribute URIs and attribute values of the object.
-     *
-     * @param attributes
-     * @param scimObjectType
-     * @return
-     */
-    public static SCIMObject constructSCIMObjectFromAttributes(Map<String, String> attributes,
-                                                               int scimObjectType)
-            throws CharonException, NotFoundException, BadRequestException {
+    public static SCIMObject constructSCIMObjectFromAttributes(UserManager userManager, Map<String, String> attributes,
```

Comment:
don't change public methods. use overloaded methods instead

Reply:
It is a overloaded method. You can find the previous method here. https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/blob/e9cd1c6e31066bcb2d3764a59d851688c05f746d/components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/utils/AttributeMapper.java#L384.